### ISSUE TITLE
Add "Open KeePassXC" to Auto-Type dialog

### ISSUE THREAD

**Issue by mrpg** [keepassxreboot/keepassxc#2057]
First of all: Thanks for your amazing work. Please grant me permission to humbly offer you a feature request.

## Expected Behavior/Current Behavior

When pressing the Auto-Type shortcut, KeePassXC currently either shows

- a dialog from which the matching account can be selected (if Always ask is activated) or
- a dialog which states that "Couldn't find an entry that matches the window title".

It would be very useful if in these two dialogs, there was a button that would allow us to open KeePassXC itself and, presumably, manually copy the password & username for an account that was not automatically selected by Auto-Type (perhaps because the window title is not informative).

## Possible Solution
Could be implemented in Qt by adding a button in the above dialogs that redirects to the main window of KeePassXC. (I'd add it myself, but sadly I don't know anything about Qt development.)

## Context
Some websites do not have a meaningful <title>, and therefore KeePassXC does not find a match. Adding a button to open the main window of KeePassXC would remove the otherwise required step to manually open KeePassXC from the system tray (i.e., use the mouse) and allow users to enter the login credentials more quickly.

## Debug Info
KeePassXC - Version 2.3.3
Revision: 0a155d8

Libraries:
- Qt 5.11.0
- libgcrypt 1.8.3

Operating system: Linux 4.16.13-2-zen
CPU architecture: x86_64
Kernel: linux 4.16.13-2-zen

Enabled extensions:
- Auto-Type
- Browser Integration
- Legacy Browser Integration (KeePassHTTP)
- SSH Agent
- YubiKey

**Comment by TheZ3ro** [comment]
Sincerely I don't see much utility in this. 
You can use https://github.com/cloutierjo/titleUrl et similar to have the website url in the title.
Autotype url matching also supports regexp to better match the window title.

If you know a specific program/window that don't display anything useful in the title don't even trigger autotype and use KeePassXC directly

**Comment by droidmonkey** [comment]
This is similar to the suggestion to have a global shortcut that opens KeePassXC (my preferred route).

**Comment by hifi** [comment]
Personally I use this very simple WebExtension for the URL: https://addons.mozilla.org/en-US/firefox/addon/append-domain/?src=search

**Comment by mrpg** [comment]
Adding a global shortcut would indeed be another solution, but I believe it has been discussed here and it won’t be implement due to it requiring massive code changes.

The browser add-on is not a bad idea. However, adding a button to the Auto-Type dialog should be the superior solution as it should require only minimal changes in the KeePassXC sources and it would not be necessary to install third party add-ons in browsers and other applications.

Some of us mainly use the keyboard, be it for health/personal reasons or because our window manager prefers that mode of operation. Manually starting KeePassXC using the mouse requires a change from the keyboard to the mouse, which is not optimal and can actually be quite exhausting!

I may try to add such a button to KeePassXC myself, but you are correct that indeed, a browser add-on for now solves a lot of my issues.

Thanks!

**Comment by TheZ3ro** [comment]
Better automate things with a browser add-on instead of clicking a button every time a website don't match autotype.

Anyway I contributed to the add-on I linked before (https://github.com/cloutierjo/titleUrl/graphs/contributors) so not really third party :stuck_out_tongue:

**Comment by droidmonkey** [comment]
If this is for accessibility we can definitely look into it

**Comment by droidmonkey** [comment]
Covered by #5864 and #99
